internal/samsahai/exporter/metric.go

```diff
@@ -10,6 +11,16 @@ import (
 	s2hlog "github.com/agoda-com/samsahai/internal/log"
 )
 
+type ActivePromotionMetricState string
+
+const (
+	Waiting ActivePromotionMetricState = "waiting"
+	Deploying ActivePromotionMetricState = "deploying"
+	Testing ActivePromotionMetricState = "testing"
+	Promoting ActivePromotionMetricState = "promoting"
+	Destroying ActivePromotionMetricState = "dsetroying"
+
+)
```

Comment:
if it's being used in this package only, should we use lower case to be private?